### PR TITLE
sqrtm - throw an error for matrices with dimension greater than two

### LINKED ISSUE #1597: sqrtm of 3D matrix returns undefined
Try `math.sqrtm(math.zeros(2, 2, 2))` in mathjs v6. The function will return `undefined` - it should either return something sensible or throw an error.

### PR DESCRIPTION
**Pull request by KonradLinkowski**
Fixes #1597
Just used this https://github.com/josdejong/mathjs/issues/1597#issuecomment-522934657

### CODE REVIEW

File: src/function/matrix/sqrtm.js [hunk 1]
```diff
@@ -65,6 +65,7 @@ export const createSqrtm = /* #__PURE__ */ factory(name, dependencies, ({ typed,
   return typed(name, {
     'Array | Matrix': function (A) {
       const size = isMatrix(A) ? A.size() : arraySize(A)
+      console.log(A, size, size.length)
```
Comment (josdejong) [inline src/function/matrix/sqrtm.js:68]: Can you clean up this `console.log`?

**Comment by josdejong** [comment]
Thanks Konrad, looks good 👍 ! I made one small comment in the code, can you look into that?

File: src/function/matrix/sqrtm.js (hunk 1 above)
Comment (KonradLinkowski) [reply src/function/matrix/sqrtm.js]: I'm a little ashamed that I forgot about that.
It's fixed now

**Comment by KonradLinkowski** [comment]
It's done I guess

File: src/function/matrix/sqrtm.js (hunk 1 above)
Comment (josdejong) [reply src/function/matrix/sqrtm.js]: 😂 happens to me too every now and then

**Comment by josdejong** [comment]
Thanks for the update, I've merged your PR now (not yet published)

**Pull request merged**